lib/elixir/lib/file.ex

```diff
@@ -634,10 +634,10 @@ defmodule File do
   `destination`. If the source is a directory, it copies
   the contents inside source into the destination.
 
-  If a file already exists in the destination,
-  it invokes a callback which should return
-  `true` if the existing file should be overwritten,
-  `false` otherwise. The callback defaults to return `true`.
+  If a file already exists in the destination, it invokes `callback`.
+  `callback` must be a function that takes two arguments: `source` and `destination`,
+  which should return `true` if the existing file should be overwritten, and `false` otherwise.
+  The callback defaults to return `true`.
```

Comment:
The punctuation looks weird here in the new sentence. What about

> `callback must be a function that takes two arguments: `source` and `destination`. The callback should return `true` if the existing file should be overwritten and `false` otherwise.

?